.github/workflows/images-build.yaml

```diff
@@ -0,0 +1,35 @@
+name: Build images
+
+on:
+  push:
+    branches:
+      - 'main'
+      - 'release*'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions: 
+  contents: read
+
+jobs:
+  build-images:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+      - name: Setup build env
+        uses: ./.github/actions/setup-build-env
+        with:
+          build-cache-key: build-images
+      - name: ko build
+        run: make ko-build-all
```

Comment:
Will all three image builds run in parallel?

Reply:
Nope

Reply:
But it will be faster because we will have better caching

Reply:
Ah you are right!